contents/blog/remote-culture.md

```diff
@@ -0,0 +1,238 @@
+
+## Communication 
+
+<br />
+
+![GitHub Chat](../images/blog/remote-culture/github-chat.png) 
+
+<br />
+
+Communication lies at the heart of just about everything concerning remote work.
+
+Since you can't directly see people and talk to them in person, it's essential that you have great communication, both from a perspective of "workplace" wellbeing, as well as productivity. 
+
+Communication is a broad topic, but here are three points that are particularly important to us:
+
+#### Asynchronous Communication
+If possible, structure your company to use as much asynchronous communication as possible. 
+
+This means people shouldn't rely on getting an immediate response from someone else in order to do their work. In our case, most of our communication happens openly on GitHub. This is great because each conversation occurs within a specific context (e.g. of an issue or PR) and all the information needed to discuss the topic should be readily available within that context. Additionally, people aren't pressed to reply immediately, and will get to it when they get to it (sometimes with a little nudge).
+
+At PostHog, for example, we don't send internal emails. 99% of the time, we're talking on GitHub, Slack, or a call. We also mostly use Slack for non-work-related, private, and more urgent conversations. If it doesn't fit those categories, chances are it could be on GitHub.
+
+#### Transparency
+Transparency means that if some information or a particular conversation can be public, we should try to make it public. 
+
+This way, we create an ever-growing record of things that have been previously discussed, questions that have been asked, and answers that have been found. 
+
+As such, when someone needs some info, they don't need to message a team member to get what they need. They can just find it immediately. 
+We extend this concept quite deeply, having our entire [Company Handbook](/handbook) public, which outlines things like how we do sales calls, our approach to talking to investors, and how we hire people. As such, if there's ever a question about how we operate as a company, referring to the handbook will usually get you the answer. 
+
+Additionally, our massive backlog of issues and pull requests also contains very useful information regarding how we've come to certain decisions, from engineering to marketing. 
+
+#### Written Communication
```

Comment:
I think this could be made way more important.

Some stuff here is kind of generic - like standups/1:1s. They obviously are important but they're not unique.

However, WRITE STUFF DOWN! is a key thing that has a big impact. Maybe cut out the generic policies and focus on the special ones?